pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/ModuleRenderingServletTest.java

```diff
@@ -19,11 +20,10 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.*;
-import static org.mockito.Mockito.lenient;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
 
 @ExtendWith({SlingContextExtension.class, MockitoExtension.class})
-public class AsciidocRenderingServletTest {
+public class ModuleRenderingServletTest {
```

Comment:
Same here regarding class naming: `ModuleRenderingTest` (since you are testing the `ModuleRendering` class)

Reply:
Updated.